camp/sponsorships.html

```diff
@@ -85,6 +56,10 @@ <h2 id="mmm-nutella">Campsite Sponsorships</h2>
       <a href="https://www.yld.io/"><img src="../images/YLD_logo_black.svg" alt="" class="yld"></a>
       <p>Diversity sponsor, <a href="https://www.yld.io/"><strong>YLD</strong></a>, is a team of motivated and experienced individuals passionate for software engineering and dream of making digital transformation possible for everyone.</p>
     </div>
+    <div class="sponsor meetup">
+      <a href="https://www.meetup.com/"><img src="../images/meetup.png" alt="" class="meetup"></a>
+      <p>Pool Party sponsor, <a href="https://www.yld.io/"><strong>Meetup</strong></a>, is the world's largest network of local groups. Meetup makes it easy for anyone to organize a local group or find one of the thousands already meeting up face-to-face.</p>
```

Comment:
I think this url should be [https://www.meetup.com/](https://www.meetup.com/) and not [yld.io](https://www.yld.io/).